style.css

```diff
@@ -6,7 +6,7 @@
 .main {
   background-color: #f7f7f9;
   width: 375px;
-  height: 5410px;
+  height: 5810px;
   padding-top: 19px;
   padding-left: 0;
   display: grid;
```

Comment:
- [ ] [_OPTIONAL_]setting fixed width and height is creating an unnecessary space on your site better to use flexible units, Kindly style the section so that it looks similar to the design.

Your Implementation | Figma Design
:---: | :---:
![image](https://user-images.githubusercontent.com/59416136/154267506-ff7ec26a-71f3-4260-804d-429f463d15c7.png) | ![image](https://user-images.githubusercontent.com/59416136/154267739-0a1efab3-6b86-43f1-9c8e-2a9eae07f197.png)

Kindly fix it.